conversations/tool_handling.go

```diff
@@ -28,6 +29,19 @@ func (c *Conversations) HandleToolCall(userID string, post *model.Post, channel
 		return err
 	}
 
+	// Get the root post ID for conversation-scoped permissions
+	rootPostID := post.RootId
+	if rootPostID == "" {
+		rootPostID = post.Id
+	}
+
+	// Handle permission updates
+	if autoApproveTool != "" {
```

Comment:
Why is this handling permissions updates? Isn't this handle by a separate endpoint?